rtlib/src/DisegniSegmenter.cpp

```diff
@@ -4,15 +4,27 @@
 #include <limits>
 #include <map>
 #include <set>
+#include <iostream>
 
 #include <opencv2/imgproc.hpp>
+#include <opencv2/core/core.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <itkOpenCVImageBridge.h>
+#include <rt/ImageTypes.hpp>
+#include <itkScalarImageToHistogramGenerator.h>
+#include <itkBinaryThresholdImageFilter.h>
+#include <itkOtsuMultipleThresholdsCalculator.h>
+#include <itkOtsuMultipleThresholdsImageFilter.h>
+#include <itkRescaleIntensityImageFilter.h>
+#include <highgui.h>
```

Comment:
Review the [Header Includes](https://github.com/viscenter/style-guides/blob/master/C%2B%2B%20Style%20Guide.md#header-includes) section of the style guide and update accordingly.